gatorgrade/output/tests/test_function.py

```diff
@@ -0,0 +1,66 @@
+""" Tests to ensure the output_functions.py functions work properly. """
+
+import pytest
+from colorama import init, Fore
+from output import output_functions
+
+init()
```

Comment:
Could you leave a comment or another indication of what init() does here? Looking at it right now I'm not entirely sure what its purpose is, so having info on it would be helpful!